### ISSUE TITLE
Search previous occurence does not work

### ISSUE THREAD

**Issue by doubleloop** [gnunn1/terminix#250]
Just installed (0.57.0), and noticed that search works only one way, you can search forward but I could not search backward (both shortcut and icon with up arrow).

Also It would be nice to be able to highlight all occurrences.

**Comment by arshavindn** [comment]
The lastest build from git works for me but it does not find all match words

**Comment by doubleloop** [comment]
Yes, after testing little more, I found out that even forward search does not find all occurences 

for example I can not find all occurrences of  `master` in console state:
```bash
doubleloop@codi-floydd ~/src/terminix  (master|…) 
$ ls
data          dub.selections.json  LICENSE    source
dscanner.ini  extract-strings.sh   po         terminix
dub.json      install.sh           README.md  uninstall.sh
doubleloop@codi-floydd ~/src/terminix  (master|…) 
$ ls    
data          dub.selections.json  LICENSE    source
dscanner.ini  extract-strings.sh   po         terminix
doubleloop@codi-floydd ~/src/terminix  (master|…) 
$ 
doubleloop@codi-floydd ~/src/terminix  (master|…) 
$ master                                                                                              1 ↵
zsh: command not found: master
Could not find the database of available applications, run update-command-not-found as root to fix this   
master: command not found
doubleloop@codi-floydd ~/src/terminix  (master|…) 
$                                                                                                   127 ↵
doubleloop@codi-floydd ~/src/terminix  (master|…) 
$
```

**Comment by gnunn1** [comment]
@doubleloop Does your example work as expected on gnome-terminal, there is an open bug for the VTE with regards to not finding all matches but the circumstances of that bug don't quite match this case.

https://bugzilla.gnome.org/show_bug.cgi?id=694128

**Comment by doubleloop** [comment]
In Gnome Terminal on this example all occurrences are found although search backward does not work (still after last occurrence it starts from the top)

**Comment by gnunn1** [comment]
So after looking at this it appears the issue is that Terminix sets the search wrap-around to false by default while gnome-terminal sets to true by default. If I set wrap-around to true I get identical behavior between the two as far as I can tell. Same if I set both to false, I again get the same behavior, albeit buggy. Since both are based on VTE no surprises there.

I have updated Terminix to use true as the default for wrap-around so at least the initial behavior between the two will be the same and less buggy.
